fetch.bs

```diff
@@ -7217,8 +7233,9 @@ method steps are:
    <li><p><a lt=terminated for=fetch>Terminate</a> the ongoing fetch with the aborted flag set.
   </ol>
 
- <li><p>Let <var>handleFetchDone</var> be to <a>finalize and report timing</a> with
- <var>response</var>, <var>globalObject</var>, and "<code>fetch</code>".
+ <li><p>Let <var>handleFetchDone</var> given <a for=/>response</a> <var>response</var> be to
+ <a>finalize and report timing</a> with <var>response</var>, <var>globalObject</var>, and
+ "<code>fetch</code>".
```

Comment:
I think you should rename the _response_ variable here as otherwise it's confusing with the other one, no?

Reply:
ok, renaming to `responseObject`

Reply:
Oh sorry, this can be _response_ because _responseObject_ is used in the outer scope. And _response_ is used below, but only within the scope of those substeps.